README.md

```diff
@@ -91,6 +91,7 @@ This plugin intends to support linting of ES2015+ (ES6+) import/export syntax, a
 [`no-unassigned-import`]: ./docs/rules/no-unassigned-import.md
 [`no-named-default`]: ./docs/rules/no-named-default.md
 [`no-anonymous-default-export`]: ./docs/rules/no-anonymous-default-export.md
+[`no-default-export`]: ./docs/rules/no-default-export.md
```

Comment:
You should mention this under one of the headers so it is visible in the README.

Reply:
Good catch! I completely missed that.

Reply:
Fixed!